docs/modules/examples/pages/advanced-rag.adoc

```diff
@@ -0,0 +1,798 @@
+
+== Combined Techniques
+
+Finally, you can combine both the MultiQuery and ParentDocument techniques, as shown below.
+[tabs]
+======
+Python::
++
+[source,python]
+----
+multi_parent_retriever = MultiQueryRetriever.from_llm(
+    retriever=parent_retriever, llm=model
+)
+
+multi_parent_chain = (
+    {"context": multi_parent_retriever, "question": RunnablePassthrough()}
+    | prompt
+    | model
+    | StrOutputParser()
+)
+
+# run the queries
+do_retrieval(multi_parent_chain)
+----
+
+Result::
++
+[source,console]
+----
+----------------------------------------
+Question: What motivates the narrator, Montresor, to seek revenge against Fortunato?
+
+Answer: The narrator, Montresor, seeks revenge against Fortunato because
+Fortunato insulted him.
+
+Total Tokens: 1880
+
+----------------------------------------
+Question: What are the major themes in this story?
+
+Answer: The major themes in this story are revenge, deception, and the power
+of manipulation.
+
+Total Tokens: 2373
+
+----------------------------------------
+Question: What is the significance of the story taking place during the carnival season?
+
+Answer: The significance of the story taking place during the carnival season
+is that it provides a chaotic and festive atmosphere, which allows the
+narrator to carry out his revenge plot without arousing suspicion.
+
+Total Tokens: 2417
+
+----------------------------------------
+Question: How is vivid and descriptive language used in the story?
+
+Answer: Vivid and descriptive language is used in the story to create a
+detailed and immersive atmosphere. It helps to paint a clear picture
+of the setting, such as the granite walls, iron staples, and chains.
+The language also conveys the emotions and actions of the characters,
+such as the protagonist's astounded and implored tone, and the
+friend's moaning cry and furious vibrations. Additionally, the
+language describes the construction of the wall and the appearance of
+the interior recess, adding to the suspense and tension of the story.
+
+Total Tokens: 1394
+
+----------------------------------------
+Question: Is there any foreshadowing in the story? If yes, how is it used in the story?
+
+Answer: Yes, there is foreshadowing in the story. The foreshadowing is used to
+hint at the fate of Fortunato and the narrator's plan for revenge. The
+mention of the chains, padlock, and walling up the entrance of the
+niche all foreshadow the narrator's intention to trap and bury
+Fortunato alive. Additionally, the mention of the Amontillado wine and
+the narrator's comment about Fortunato's cough hint at the method of
+Fortunato's demise.
+
+Total Tokens: 2546
+----
+======
+
+This is by far the most expensive technique, but perhaps returns the best results.
+
+== Cleanup
+
+Astra has a maximum number of collections per DB.
+Delete the collection after you are done exploring this example.
+
+If this call returns a timeout, wait a few seconds and try again.
```

Comment:
per Enrico, lets omit this warning about the timeout from our docs. Hopefully this will get fixed on the Astra side soon.